packages/frontend/src/__tests__/onBoarded.js

```diff
@@ -0,0 +1,35 @@
+import { screen, render } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import Onboarded from '../pages/startPage/onboarded'
+
+const mockHistoryPush = jest.fn()
+
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    useHistory: () => ({
+        push: mockHistoryPush,
+    }),
+}))
+
+test('should render component properly', () => {
+    const title = '🎉 You’re in!'
+    const rep = '30 Rep + 3 personas await you!'
+    const p1 =
+        'Excllent! One huge difference of UniRep Social is that you don’t have to interact with any wallet, we have come up with this solution to smooth out the experience. If you are interested, you can learn more from our developer document.'
```

Comment:
The typo in the word excellent should be fixed. This can be updated in the component itself as well.